pkg/api/admin/config.go

```diff
@@ -103,6 +103,8 @@ type ImageConfig struct {
 	GenevaTDAgent *string `json:"genevaTDAgent,omitempty"`
 	GenevaStatsd  *string `json:"genevaStatsd,omitempty"`
 	MetricsBridge *string `json:"metricsBridge,omitempty"`
+
+	MonitorAgent *string `json:"monitorAgent,omitempty"`
```

Comment:
It might be easier to understand if this were renamed to `AzureMonitorAgent` or `LogAnalyticsAgent`